creation/web_base/condor_startup.sh

```diff
@@ -150,6 +151,12 @@ cat > "$condor_job_wrapper" <<EOF
 
 # This script is started just before the user job
 # It is referenced by the USER_JOB_WRAPPER
+# /bin/sh is used for Busybox compatibility (small images)
+if [ -n "$BASH_VERSION" ]; then
```

Comment:
Is the single square bracket notation used here to ensure this line of code is POSIX-compliant and not just bash (instead of double square bracket)?